dev/AppNotifications/WICUtility.h

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#pragma once
+
+#include "pch.h"
+
+namespace Microsoft::Windows::AppNotifications::WICHelpers
+{
+    void WriteHIconToPngFile(wil::unique_hicon const& hIcon, _In_ PCWSTR pszFileName);
```

Comment:
All Shell helpers should be moved here. This function should be private. So only include its definition and implementation in the cpp.

Reply:
Per @DrusTheAxe comment above,  this should be called ShellLocalization.h